packages/react-components/react-breadcrumb/src/components/BreadcrumbButton/useBreadcrumbButton.ts

```diff
@@ -17,11 +17,17 @@ export const useBreadcrumbButton_unstable = (
   ref: React.Ref<HTMLButtonElement | HTMLAnchorElement>,
 ): BreadcrumbButtonState => {
   const { appearance, iconPosition, size } = useBreadcrumbContext_unstable();
+  const { current = false, icon, ...rest } = props;
 
-  const { current = false, ...rest } = props;
   return {
     ...useButton_unstable(
-      { ...rest, appearance: props.appearance || appearance, iconPosition: props.iconPosition || iconPosition },
+      {
+        ...rest,
+        appearance: props.appearance || appearance,
+        iconPosition: props.iconPosition || iconPosition,
+        icon,
+        'aria-current': current ? 'page' || props['aria-current'] : undefined,
```

Comment:
Same here,  `props['aria-current']` should come first so it's used if defined
```suggestion
        'aria-current': current ? props['aria-current'] ?? 'page' : undefined,
```

Reply:
copy-past issue, thanks for catching that